packages/headless/src/features/facets/facet-search-set/facet-search-reducer-helpers.ts

```diff
@@ -97,7 +97,7 @@ export function handleFacetSearchFulfilled<T extends FacetSearchResponse>(
 export const defaultFacetSearchOptions: FacetSearchRequestOptions = {
   captions: {},
   numberOfValues: 10,
-  query: '',
+  query: '**',
```

Comment:
What happens if a user types, then clears the text? Wouldn't the query become an empty string leading to an inconsistent experience?

Another thought perhaps for @acote-coveo, would it make sense to have this be the default on the search-api? Or is there a benefit in treating an empty string as an empty string?

Assuming we are keeping the default client-side, let's update the ts-doc annotations to reflect the new value?

Reply:
What happens if a user types, then clears the text? Wouldn't the query become an empty string leading to an inconsistent experience?

Update text always prepends and appends the wildcards, like I mentioned in the description.

@acote-coveo  I've mentionned this issue in Slack for the V4 Facet search endpoint improvements, think it can be changed for the V2 as well?

Reply:
I will investigate that as part of my V4 facet search endpoint. Thanks for the tips guys